doctr/models/detection/differentiable_binarization.py

```diff
@@ -40,10 +47,10 @@ class DBPostProcessor(DetectionPostProcessor):
     """
     def __init__(
         self,
-        unclip_ratio: Union[float, int] = 1.5,
+        unclip_ratio: Union[float, int] = 2.,
         min_size_box: int = 5,
-        max_candidates: int = 100,
-        box_thresh: float = 0.5,
+        max_candidates: int = 1000,
```

Comment:
I need to investigate for that but empirically it seems to be a good bunch of parameters